documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -3,43 +3,60 @@
 // assembly-configuring-kafka-listeners.adoc
 
 [id='proc-accessing-kafka-using-ingress-{context}']
-= Accessing Kafka using ingress
+= Accessing Kafka using ingresses
```

Comment:
TBH, I'm not sure this sounds right. The access is facilitated through a single Ingress controller or multiple Ingress resources. So maybe we can put there either `Accessing Kafka using Ingress controller` or `Accessing Kafka using ingress resources`?

Reply:
Right. Let's say _Accessing Kafka using an Nginx Ingress Controller_

Reply:
`Accessing Kafka using a Kubernetes Nginx Ingress Controller` ;-)